functions/backup.bash

```diff
@@ -180,6 +180,8 @@ create_amanda_config() {
   if ! (echo -e "amgtar:gnutar_path=$GNUTAR\\n" >> "$amandaSecurityConf"); then echo "FAILED (amanda-security.conf)"; return 1; fi
   if ! cond_redirect chown --recursive "${backupUser}:backup" "$amandaHosts" "$configDir" "$databaseDir" "$indexDir" /var/log/amanda; then echo "FAILED (chown)"; return 1; fi
   if [[ $config == "openhab-dir" ]]; then
+    if ! cond_redirect sudo -u "${backupUser}" "touch ${storageLoc}/TEST"; then echo "FAILED (storage write access for user backup)"; return 1; fi
+    rm -f "${storageLoc}/TEST"
```

Comment:
Wouldn't we want this after we change the permissions of the backup user, ie move to after line 186?

Reply:
right, I now combined those